### DISCUSSION TITLE
FastAPI on pypy

### DISCUSSION THREAD

**Discussion by DragonovAk47** [fastapi/fastapi#3944] [Questions] [answered]
Will pypy make fastapi more fast??
I am new to pypy but what according to what i am reading  scripts containing heavy loops and numerical calculation only can benefit from pypy.
So am intrigued to know what effects will it have on fastapi

**Comment by ghandic** [comment, marked as answer]
It depends on your use case. If you have lots of numerical calculations then pypy will be faster as it used JIT. However, it would not be faster for IO-bound problems.

I would suggest moving your computations outside of Python rather than using pypy. Make full use of numpy and numba for instance and let the optimized C code do the heavy lifting.
